components/PlaylistDetails.tsx

```diff
@@ -32,11 +32,14 @@ const PlaylistDetails: React.FC<Props> = ({ playlist, isLoading }) => {
       <div className="text-center md:text-left md:ml-5">
         <Heading level="h1">{playlist.name}</Heading>
         <h2 className="text-gray-400 text-xl mt-2">{playlist.description}</h2>
+        <h3 className="text-gray-400 text-base">
+          {playlist.tracks.total}{' '}
+          {playlist.tracks.total > 1 ? 'Tracks' : 'Track'}
```

Comment:
I think the logic here should actually be `playlist.tracks.total === 1 ? 'Track' : 'Tracks'` because currently if there are 0, it will say "0 Track" which should be plural